elastic/kibana
# [ILM] Removed `stack_trace` usage from ILM extension for Index Management

Fixes https://github.com/elastic/kibana/issues/127190

## Summary

This PR updates ILM extension for Index Management to not use `stack_trace` property because it was removed from ES response in https://github.com/elastic/elasticsearch/pull/84266. There is now no button in UI to display ILM error's stack trace. 

### Screenshots 
#### Before 
<img width="2298" alt="Screenshot 2022-03-23 at 17 19 49" src="https://user-images.githubusercontent.com/6585477/159750503-b37ad93b-1e58-4fd8-ba71-d14567726e86.png">

#### After

<img width="2292" alt="Screenshot 2022-03-23 at 17 39 46" src="https://user-images.githubusercontent.com/6585477/159750902-3276b32a-ee72-4b2b-aee0-87165e8daaab.png">

## How to test
1. Create a test index in Dev Tools Console 
```
POST /test_index/_doc
{
  "test": "test"
}
```
2. Create a test ILM policy with a maximum of 1 document for rollover in Dev Tools Console 
```
PUT _ilm/policy/test1
{
  "policy": {
    "phases": {
      "hot": {
        "min_age": "0ms",
        "actions": {
          "set_priority": {
            "priority": 100
          },
          "rollover": {
            "max_primary_shard_size": "50gb",
            "max_age": "30d",
            "max_docs": 1
          }
        }
      },
      "warm": {
        "min_age": "0nanos",
        "actions": {
          "set_priority": {
            "priority": 50
          }
        }
      }
    }
  }
}
```
3. Update cluster settings to not wait too long for rollover check
```
PUT /_cluster/settings
{
  "transient": {
    "logger.org.elasticsearch.xpack.core.indexlifecycle": "TRACE",
    "logger.org.elasticsearch.xpack.indexlifecycle": "TRACE",
    "logger.org.elasticsearch.xpack.core.ilm": "TRACE",
    "logger.org.elasticsearch.xpack.ilm": "TRACE",
    "indices.lifecycle.poll_interval": "10s"
  }
}
```
4. Add ILM to index settings in Dev Tools Console
```

PUT /test_index/_settings
{
  "index": {
      "lifecycle": {
        "name": "test1"
      }
  }
}
```
5. Add more documents to the test index with several requests in Dev Tools Console
```
POST /test_index/_doc
{
  "test": "test"
}
```
6. In Index Management, find the test index and open details flyout. Wait until an ILM error is displayed in the flyout. 

### Checklist
